src/Models/Message.php

```diff
@@ -114,11 +168,15 @@ public function getAddressee(): CharacterInterface
     
     /**
      * Sets the author of this motd
```

Comment:
I'm confused by this. Is this a message or an MotD? Why would MotD have an addressee/recipient? It's a broadcast, right?

Also, since we really don't want users to modify messages once they're sent, why provide mutators for author/addressee/title/body? Immutability makes for easier reasoning and safer code---you can only set those via the constructor.